t/19-isotovideo-command-processing.t

```diff
@@ -394,6 +394,34 @@ subtest signalhandler => sub {
     } qr/isotovideo received signal INT/, 'Signal logged';
     is($last_signal, 'INT', 'Event emitted');
 };
+subtest 'Check exit_code_from_test_results' => sub {
+    my $runner = OpenQA::Isotovideo::Runner->new;
+    use Mojo::File 'path';
+    use Mojo::JSON 'encode_json';
```

Comment:
Please add `use` statements only at the top of the file (preferably in alphabetical order and merging with existing `use` statements).